src/test/java/com/fasterxml/jackson/dataformat/xml/ser/Jdk8StreamSerialization302Test.java

```diff
@@ -0,0 +1,151 @@
+
+        @JacksonXmlElementWrapper(localName = "elements")
+        @JacksonXmlProperty(localName = "element")
+        public Iterator<String> getData() {
+            return data;
+        }
+
+        public void setData(Iterator<String> data) {
+            this.data = data;
+        }
+    }
+
+    public void testCollectionSerialization() throws JsonProcessingException {
+        Collection<String> list = new ArrayList<>();
+        list.add("a");
+        list.add("b");
+
+        assertEquals("<ArrayList><item>a</item><item>b</item></ArrayList>",
+            OBJECT_MAPPER.writeValueAsString(list));
+    }
+
+    public void testListSerialization() throws JsonProcessingException {
+        List<String> list = new ArrayList<>();
+        list.add("a");
+        list.add("b");
+        assertEquals("<ArrayList><item>a</item><item>b</item></ArrayList>",
+            OBJECT_MAPPER.writeValueAsString(list));
+    }
+
+    public void testListIteratorSerialization() throws JsonProcessingException {
+        List<String> list = new ArrayList<>();
+        list.add("a");
+        list.add("b");
+        Iterator<String> listItr = list.iterator();
+
+        assertEquals("<Itr><item>a</item><item>b</item></Itr>",
+            OBJECT_MAPPER.writeValueAsString(listItr));
+    }
+
+
+    public void testStreamIteratorSerialization() throws JsonProcessingException {
+        assertEquals("<Adapter><item>a</item><item>b</item></Adapter>",
+            OBJECT_MAPPER.writeValueAsString(Stream.of("a", "b").iterator()));
+    }
+
+    // [dataformat-xml#329] : Jackson ignores JacksonXmlElementWrapper on Stream
+    public void testCollectionWrapperSerialization329() throws JsonProcessingException {
+        Collection<String> collection = new ArrayList<>();
+        collection.add("a");
+        collection.add("b");
+        CollectionWrapper329 wrapper = new CollectionWrapper329();
+        wrapper.setData(collection);
+
+        assertEquals(
+            "<CollectionWrapper329><elements>" +
+                "<element>a</element>" +
+                "<element>b</element>" +
+                "</elements></CollectionWrapper329>",
+            OBJECT_MAPPER.writeValueAsString(wrapper));
+    }
+
+    // [dataformat-xml#329] : Jackson ignores JacksonXmlElementWrapper on Stream
+    public void testIteratorWrapperSerialization329() throws JsonProcessingException {
+        Collection<String> collection = new ArrayList<>();
+        collection.add("a");
+        collection.add("b");
+        IteratorWrapper329 wrapper = new IteratorWrapper329();
+        wrapper.setData(collection.iterator());
+
+        assertEquals(
+            "<IteratorWrapper329><elements>" +
+                "<element>a</element>" +
+                "<element>b</element>" +
+                "</elements></IteratorWrapper329>",
+            OBJECT_MAPPER.writeValueAsString(wrapper));
+    }
+    
+    /*
+    /**********************************************************
```

Comment:
can you not add a test scope dependency on Jdk8Module ?

Reply:
Ideally this should not be done, to keep cross-repo dependencies minimal.